client/src/components/PersonCard/PersonCard.js

```diff
@@ -29,7 +47,12 @@ PersonCard.defaultProps = {
     { url: "#", interests: "budgeting" },
     { url: "#", interests: "travel" },
     { url: "#", interests: "outdoors" }
-  ]
+  ],
+  dot: <div className="PersonCard__online"></div>,
```

Comment:
What's this doing? Shouldn't the prop be something like `status` or `online`? Like a boolean value that controls whether or not the dot is visible